.github/workflows/tests-and-lint.yml

```
@@ -0,0 +1,266 @@
+  pull_request:
+    branches: [ main ]
+    paths:
+      - 'projects/**'
+      - 'shared/**'
+      - 'tests/**'
+      - 'scripts/**'
+      - 'package.json'
+      - 'package-lock.json'
+      - 'eslint.config.js'
+      - '.stylelintrc.json'
+    paths-ignore:
```

actionlint (1.7.12)

[error] both "paths" and "paths-ignore" filters cannot be used for the same event "pull_request". note: use '!' to negate patterns

(events)